src/dashboard/src/pages/Cluster/Workers.tsx

```diff
@@ -113,7 +109,7 @@ const Workers: FunctionComponent<Props> = ({ data: { config, types, workers }, o
     }];
     columns.push(...resourceColumns);
     columns.push({
-      title: 'GPU Utilization',
+      title: 'Available GPU Utilization',
```

Comment:
Can we keep "GPU Utilization" for worker nodes? Whether the GPUs are assigned is a concept from our service.

Reply:
done